docs/community.md

```diff
@@ -2,7 +2,7 @@
 
 Welcome to the FIDO Device Onboard community. FIDO Device Onboard is an open source project sponsored by [LF Edge](https://www.lfedge.org/projects/fidodeviceonboard/).
 
-Community practices are documented at the [Secure Device Onboard wiki](https://wiki.lfedge.org/display/SDO/Secure+Device+Onboard).
+Community practices are documented at the [Secure Device Onboard wiki](https://wiki.lfedge.org/display/LE/Secure+Device+Onboard).
```

Comment:
Replace it with the respective FDO link.

Reply:
https://github.com/fido-device-onboard/docs-fidoiot/pull/63/commits/cde3715f015092abe999b2fc63b1936d78b2e939